### PR TITLE
feat: open-source Sana-Video 2.0

### PR DESCRIPTION
**Pull request by lawrence-cj**
## Summary

- add the production SANA-Video 2.0 architecture with public 5B and 14B variants
- reuse the shared `train_video_scripts/train_video_ivjoint_chunk.py` and `inference_video_scripts/inference_sana_video.py` entry points with minimal SANA-Video 2.0 compatibility
- add clean 480p and 720p reference configs for the 5B variant; the 14B config and checkpoint are intentionally not included yet
- release the jointly post-trained T2V/TI2V [SANA-Video 2.0 5B 720p checkpoint](https://huggingface.co/Efficient-Large-Model/SANA-Video_2.0_5B_720p)
- add a verified 5B 720p release demo generated from the public `hf://` checkpoint, together with its exact prompt and launch command
- add the SANA-Video 2.0 guide, model-zoo entry, README/news links, legacy asset-doc mirror, and focused tests
- keep the public surface limited to release artifacts without internal experiments, cluster settings, private-data paths, or redundant version-specific runners

## Architecture

- **5B:** 32 layers, hidden size 2560, 4 Attention Residual blocks, 4.47B parameters
- **14B:** 40 layers, hidden size 4096, 5 Attention Residual blocks, 14.25B parameters
- both variants use the final hybrid gated linear/softmax attention design and shared timestep-independent Attention Residual blocks

## Checkpoint

- HF repo: [Efficient-Large-Model/SANA-Video_2.0_5B_720p](https://huggingface.co/Efficient-Large-Model/SANA-Video_2.0_5B_720p)
- path: `checkpoints/SANA_Video_2.0_5B_720p.pth`
- SHA256: `7e557554540b4cbbc515166b43a7d307285ab250cedfb48627878227a722d25a`
- contains only the merged model `state_dict` (797 tensors); optimizer, scheduler, training state, and standalone LoRA tensors are excluded
- the model card, matching release config, Apache-2.0 license, SANA-Video collection entry, and verified demo are published

## Verified 5B 720p demo

[SANA-Video 2.0 5B release demo: cartoon rooster in a floral vintage room](https://huggingface.co/datasets/Efficient-Large-Model/Sana-assets/resolve/main/Video2/assets/release-demo/sana_video2_5b_720p_rooster.mp4)

[▶ Watch or download the generated MP4](https://huggingface.co/datasets/Efficient-Large-Model/Sana-assets/resolve/main/Video2/assets/release-demo/sana_video2_5b_720p_rooster.mp4)

- output: H.264, 1280 × 736, 193 frames, 24 FPS, 8.04 seconds
- video SHA256: `e99a8e48ace067ac3f582e9ffeb9b0a9f42af7f0f8c0cec2d74054a5088397ce`
- assets commit: [`82b9f1da`](https://huggingface.co/datasets/Efficient-Large-Model/Sana-assets/commit/82b9f1da48b5d2ed902845f14de5a18df3881670)

> **Prompt:** In a cozy, vintage room adorned with floral wallpaper, a cartoon
> rooster sits comfortably in a floral-patterned armchair, sipping from a bottle
> of beer. The rooster, with its vibrant red comb and wattle, displays a range of
> expressions—smiling, nodding, and opening its beak wide in a cheerful manner.
> The setting includes wooden furniture and another beer bottle on the table,
> adding to the relaxed atmosphere. The camera captures the rooster from a
> close-up angle, emphasizing its animated movements and lively demeanor.

Exact command used inside the one-H100 allocation (identical in the root README, release docs, and model card):

```bash
bash inference_video_scripts/inference_sana_video.sh \
  --np 1 \
  --config configs/sana_video2/SanaVideo2_5B_720p.yaml \
  --model_path hf://Efficient-Large-Model/SANA-Video_2.0_5B_720p/checkpoints/SANA_Video_2.0_5B_720p.pth \
  --txt_file=asset/samples/sana_video2_5b_720p_demo.txt \
  --cfg_scale 8 \
  --flow_shift 12 \
  --step 50 \
  --fps 24 \
  --motion_score 20 \
  --seed 0 \
  --work_dir output/sana_video2_t2v_720p_demo
```

## Impact

This exposes the model architecture, training and inference paths, release documentation, and the first SANA-Video 2.0 checkpoint. The released 5B model supports 720p, approximately 8-second text-to-video and text-image-to-video generation. The 14B architecture remains public, while its config and checkpoint remain unreleased.

## Validation

- strict merge verification: all 256 LoRA target weights match the merge formula and every non-LoRA tensor remains exact
- release checkpoint verification: exact key order/coverage, tensor metadata/value equality, single `state_dict` container, and local/remote SHA256 match
- H100 BF16 smoke test: strict 797-tensor load and finite forward output passed (8.41 GiB peak allocated)
- end-to-end H100 inference from the public `hf://` checkpoint passed with the exact 50-step command above; sampling, VAE decode, and MP4 encoding completed successfully
- generated-video verification: 193 decoded frames, 24 FPS, 1280 × 736, 8.04 seconds, exact local/remote asset SHA256 match, and visual inspection of the first/middle/final frames passed
- released LTX 2.3 VAE compatibility was fixed for Diffusers 0.38.0; the 726,105,969-parameter VAE then loaded completely in BF16
- `pytest -q tests/test_sana_video2.py` — 10 passed
- both shared entry points passed their `--help` startup checks
- MkDocs build passed; warnings remain limited to seven pre-existing issues outside the SANA-Video 2.0 page
- full `pre-commit run --all-files` passed
- privacy, cluster-path, secret-pattern, stale-reference, and diff-integrity scans passed

### CODE REVIEW

**Review by HeliosZhao — APPROVED** [review]
LGTM!
